pages/_app.js

```diff
@@ -18,6 +31,13 @@ export default function App({ Component, pageProps }) {
 
   const [images, setImages] = useState(pictures);
```

Comment:
This will probably be confusing in a couple of weeks. Maybe you can rename this to your needs. I would suggest:
```suggestion
  const [images, setImages] = useState(staticDummyDataImages);
```